### PR TITLE
Add LAUNCH_HOSTNAME

### PR DESCRIPTION
**Pull request by LDAP**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ The following environment variables are important if you don't supply a `/docker
 | `LAUNCH_PROJECT_NAME`   | `swarm-launcher`           | NO            | If you want to use a specific name for the project (similar to the stack name) |
 | `LAUNCH_SERVICE_NAME`   | random (by swarm-launcher) | NO            | If you want to use a specific name for the service |
 | `LAUNCH_CONTAINER_NAME` | random (by docker)         | NO            | If you want to use a specific name for the container (similar to the task name) |
+| `LAUNCH_HOSTNAME`       | -                          | NO            | If you want to set a specific hostname for the container
```
Comment (tlex) [inline README.md:47]: Please add ` |` at the end of the line

**Comment by tlex** [comment]
Thanks for the PR. Once you fix the line in README.md, I'll merge it and make a new release.

**Comment by LDAP** [comment]
I did force push the change to my master to preserve a clean history on yours.

**Comment by tlex** [comment]
I've released [v0.10.0](https://github.com/ix-ai/swarm-launcher/releases/tag/v0.10.0) with this feature.

Thanks again.

**Comment by LDAP** [comment]
Thanks to you. Yoir Image saved me a lot of work.

**Pull request merged**